napari/plugins/__init__.py

```diff
@@ -1,10 +1,13 @@
 import os
 import sys
+from inspect import isclass
+from typing import Callable, Dict, Sequence, Tuple, Type, Union
 
 from napari_plugin_engine import PluginManager
+from qtpy.QtWidgets import QAction, QWidget
```

Comment:
Similar to comment bellow we're getting headless mode fails. Should I put imports in `if TYPE_CHECKING` or use string for annotations?